.github/workflows/test.yaml

```diff
@@ -0,0 +1,21 @@
+name: Test
+
+on: push
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12, 14, 16]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node-version }}
```

Comment:
Can add `cache: npm`. See: https://github.com/actions/setup-node#caching-packages-dependencies

Reply:
We can't cache without package lock